vnext/ReactCommon/TEMP_UntilReactCommonUpdate/react/renderer/mapbuffer/MapBufferBuilder.cpp

```diff
@@ -33,12 +33,9 @@ MapBufferBuilder::MapBufferBuilder(uint16_t initialSize) {
 
 void MapBufferBuilder::ensureKeyValueSpace() {
   int oldKeyValuesSize = keyValuesSize_;
-  if (keyValuesSize_ >= std::numeric_limits<uint16_t>::max() / 2) {
-    LOG(ERROR)
-        << "Error: trying to assign a value beyond the capacity of uint16_t"
-        << static_cast<uint32_t>(keyValuesSize_) * 2;
-    abort();
-  }
+  react_native_assert(
```

Comment:
heads up that this might cause us to have to bring in more stuff in our lib build of ReactCommon

Reply:
They started using this macro a few weeks ago, and I already stubbed out the necessary bits of the GLog library it uses.